src/bin/pg_autoctl/pgsql.c

```diff
@@ -1187,18 +1187,18 @@ typedef struct PgsrSyncAndWALContext
 {
 	bool		parsedOk;
 	char		syncState[PGSR_SYNC_STATE_MAXLENGTH];
-	uint64_t	walLag;
+	char        latestLSN[NAMEDATALEN];
```

Comment:
A comment explaining the choice of the length here would be necessary.

Reply:
added